src/components/TalkToUs/style.js

```diff
@@ -1,25 +1,7 @@
 import { css } from "@emotion/core"
+import { mq } from "@/theme"
 
-export const container = theme => css`
-  height: 8rem;
-  width: 100%;
-  background-color: ${theme.colors.purples.normal};
-  display: flex;
-  justify-content: center;
-`
-
-export const content = css`
-  height: 100%;
-  width: 102.4rem;
-  padding-left: 1.2rem;
-  padding-right: 1.2rem;
-  box-sizing: border-box;
-  display: flex;
-  align-items: center;
-  justify-content: flex-end;
-`
-
-export const helpButton = theme => css`
+export const helpButton = (theme, opening, closing) => css`
```

Comment:
Since `opening  === !closing`, you don't need to create extra argument.

Reply:
fixed.